plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-shipping-method-block/block.tsx

```diff
@@ -178,15 +185,16 @@ const Block = ( {
 	);
 
 	return (
-		<RadioGroup
+		<div
```

Comment:
should we keep some css classes on this div that belonged to the RadioGroup?

Reply:
Are you suggesting that CSS classes are public API? The classes aren't needed for any other reason.

Reply:
In the checkout blocks they are public APIs, yes

Reply:
@ralucaStan I'm just wondering in that case how do we ever deprecate a css class? 

I'm just wondering how strictly we really adhere to this? I'm happy to do whatever is needed here, but I think it's unwise to commit to not changing any CSS classes, especially as HTML will change in structure too, if someone targets a class that is no longer a descendant we wouldn't support that would we? How do we draw a sensible line here?

Reply:
In general we kept track of classes using the prefix `.wc-block-components`; we also have some legacy classes `wc-block`. 

RadioGroup is not part of this, but I still think it should be communicated in the release announcement. @opr what do you think? Do you have more context on the CSS classes being public API?

Reply:
@ralucaStan just updating to say, As much as it pains me, I'll add whatever classes from the old wp component that we have to.

I wish we could deprecate classes somehow, but I think for now it's better to play it safe. Thanks for challenging this one.

Reply:
There was only one class to add to the div. I've added it with a comment here: https://github.com/woocommerce/woocommerce/pull/45974/commits/dd408abc341084b669632d9edfaff5a0723e40c8